unpick-definitions/global_events.unpick

```diff
@@ -0,0 +1,10 @@
+v2
+
+constant global_events net/fabricmc/yarn/constants/GlobalEvents WITHER_SPAWNS
+constant global_events net/fabricmc/yarn/constants/GlobalEvents ENDER_DRAGON_DIES
+constant global_events net/fabricmc/yarn/constants/GlobalEvents END_PORTAL_OPENED
+
+target_method net/minecraft/world/World syncGlobalEvent (ILnet/minecraft/util/math/BlockPos;I)V
+	param 0 global_events
+target_method net/minecraft/client/render/WorldRenderer processGlobalEvent (ILnet/minecraft/util/math/BlockPos;I)V
+	param 0 global_events
```

Comment:
You sure there aren't more than this (and for the other one too)?

Reply:
Pretty sure. These are all the ones handled by WorldRenderer at least...

Reply:
I was thinking maybe in the packet.

Reply:
Imo if we have missing ones we can always add them later. Indeed, the best place to check is in the packet, as there may be events that only have sound effects without render effects, for instance.

Reply:
Sound effect events are handled in WorldRenderer too
(which kinda makes the WorldRenderer name a bit inaccurate but whatever)